packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -211,6 +219,11 @@ class _InspectorTreeState extends State<InspectorTree>
     constraintDisplayController?.dispose();
   }
 
+  @override
+  void setFocus() {
+    FocusScope.of(context).requestFocus(focusNode);
```

Comment:
rename this method to requestFocus to match the flutter naming conventions.

Reply:
I tried enclosingScope and it didn't have the right behavior - I'm not sure what got scope, but it wasn't the tree. So I'm going to punt on this for now.

Reply:
Yeah, I think I was misreading your code.  What you want is `focusNode.requestFocus()`.